modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -129,6 +129,20 @@ func (h *Handler) Provision(ctx caddy.Context) error {
 	h.ctx = ctx
 	h.logger = ctx.Logger(h)
 
+	// get validation out of the way
+	for i, v := range h.Upstreams {
+		// the special cases arise with LookupSRV present, so skip the iteration if it's empty.
```

Comment:
What are "the special cases"? Maybe explain more here, or the comment can be removed.

Reply:
Active healthchecks being incompatible with lookup_srv and specifying both dial and lookup_srv. I thought it's better to explain why we're skipping the iteration.

Reply:
Now?